test/e2e/es_index_cleaner_test.go

```diff
@@ -85,5 +107,27 @@ func esIndexCleanerTest(t *testing.T, f *framework.Framework, testCtx *framework
 	if err != nil {
 		return err
 	}
-	return nil
+
+	return wait.Poll(retryInterval, timeout, func() (done bool, err error) {
+		flag, err := hasIndexWithPrefix("jaeger-")
```

Comment:
Is this needed `hasIndexWithPrefix` is also called on line 97.

Reply:
Line 97 is checking the indices exist, this part of checking they no longer exist.

Reply:
oh my bad, it looks good.